### DISCUSSION TITLE
To allow content sensitive help for blocks

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15850] [Feature improvement]
It would be helpful if we can add content sensitive popup next to blocks names.
Best

**Comment by mariuszkrzaczkowski** [comment]
give an example because I don't understand

**Comment by s2b-git** [comment]
![block-sample](https://user-images.githubusercontent.com/84110093/161428431-2a7060e5-2395-4135-b8cb-1a69e2b71b8a.jpg)
:-)
